int32 id
geometry_msgs/Pose contact_pose
float32[36] encoder_noise

